From f51f13a7e9de2b5c235a4c231138fa73d67a49b8 Mon Aug  8 13:14:18 2016
Message-Id: <f51f13a7e9de2b5c235a4c231138fa73d67a49b8@dist-git>
From: Michal Privoznik <mprivozn@redhat.com>
Date: Mon,  8 Aug 2016 13:08:51 +0200
Subject: [PATCH] RHEL: docs: Introduce subsite.xsl

https://bugzilla.redhat.com/show_bug.cgi?id=1365004

RHEL only

The file was not being distributed (now fixed in both upstream
and downstream). However, since we are building on the top of
release that is still missing the file copy it from upstream.

Signed-off-by: Michal Privoznik <mprivozn@redhat.com>
---
 docs/subsite.xsl | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)
 create mode 100644 docs/subsite.xsl

diff --git b/docs/subsite.xsl a/docs/subsite.xsl
new file mode 100644
index 0000000..108d0d8
--- /dev/null
+++ a/docs/subsite.xsl
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<xsl:stylesheet
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:exsl="http://exslt.org/common"
+  exclude-result-prefixes="xsl exsl"
+  version="1.0">
+
+  <xsl:import href="page.xsl"/>
+
+  <xsl:output
+    method="xml"
+    encoding="UTF-8"
+    indent="yes"
+    doctype-public="-//W3C//DTD XHTML 1.0 Strict//EN"
+    doctype-system="http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"/>
+
+  <xsl:variable name="href_base" select="'../'"/>
+
+  <xsl:template match="/">
+    <xsl:apply-templates select="." mode="page">
+      <xsl:with-param name="pagename" select="$pagename"/>
+    </xsl:apply-templates>
+  </xsl:template>
+
+</xsl:stylesheet>
-- 
2.8.4


